dependencies.yaml

```diff
@@ -350,7 +352,6 @@ dependencies:
         packages:
           - fmt>=10.1.1,<11
           - librmm==24.10.*,>=0.0.0a0
```

Comment:
We should probably align this librmm behavior with the libkvikio behavior (list both or list neither).

Reply:
Removed in b46bd0bdce. The `librmm` was already coming from `depends_on_librmm` everywhere that `build_cpp` was used.

Reply:
Sure, works for me! I like this `depends_on_{rapids_project}` pattern.